Semantics.md

```diff
@@ -0,0 +1,49 @@
+
+### Enumerations
+
+In general, avoid using enumerations as bit fields, instead use a set of the enumeration. This adheres to the general principle that when all else is equal, prefer typed over untyped code. A good reason to use a bit field is for its speed and small size.
+
+Bad:
+```csharp
+[Flags]
+enum Inventory { Hat = 1, Sword = 2, BigSword = 4 }
+
```

Comment:
Use bit shifting, not manual decimal math please. And you better put this on multiple lines for clarity.